Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -463,7 +468,9 @@ fragment float4 fragmentShader(RasterizerData in [[stage_in]],
 
     void Camera::Impl::Close()
     {
-        m_implData.reset(new ImplData);
+        [m_implData->cameraTextureDelegate reset];
+        m_implData.reset();
+        m_implData = std::make_unique<ImplData>();
```

Comment:
We should probably wait to make this until it's needed

Reply:
I'll leave it up to you. It's fine by me either way.

Reply:
I think I'll just leave it since the Open call assumes we have a valid pointer